test/e2e/klusterletaddonconfig_test.go

```diff
@@ -35,63 +36,39 @@ var _ = Describe("klusterletAddonConfig test for claim and hypershift cluster",
 		})
 
 		It("test klusterletAddonConfig create", func() {
-			assertManagedClusterNamespace(managedClusterName)
```

Comment:
@zhiweiyin318 @zhujian7  This part is also more clear in here: https://github.com/stolostron/klusterlet-addon-controller/blob/b37102ebae8012cacb7a1de1fe8100fff6ee2619/test/e2e/klusterletaddonconfig_test.go

Just encapsulates a general method for 3 cases to save code.